src/dropdown/dropdown.stories.ts

```diff
@@ -41,6 +41,29 @@ storiesOf("Dropdown", module)
 			onClose: action("Dropdown closed")
 		}
 	}))
+	.add("Light", () => ({
+		template: `
+		<ibm-dropdown
+			[theme]="'light'"
```

Comment:
This can just be:

```suggestion
			theme="light"
```

Reply:
But if you do it `theme="light"`, the theme will be an extra attribute appearing in the HTML Dom node. I use `[theme]="'light'"` because I think it is cleaner. However, if you guys prefer to have that as an attribute in the dom I will gladly apply your suggested changesets :-)

Reply:
Since it's just a static attr in this case I think it makes more sense to bind it as an attr ... even though it clutters the DOM a little bit ...

Reply:
Sure, just applied your change :-)